src/envoy/http/authn/authn_utils.cc

```diff
@@ -24,45 +24,68 @@ namespace AuthN {
 namespace {
 class JsonIterCallback : public Logger::Loggable<Logger::Id::filter> {
  public:
-  // The json iterator callback function
-  static bool JsonIteratorCallback(const std::string&,
-                                   const Envoy::Json::Object&,
-                                   istio::authn::JwtPayload* payload);
+  // The json iterator callback function for extracting Jwt claims
+  static bool JsonIteratorCallbackForJwtClaims(
+      const std::string&, const Envoy::Json::Object&,
+      istio::authn::JwtPayload* payload);
+  // The json iterator callback function for exracting Jwt audience
+  static bool JsonIteratorCallbackForJwtAudience(
+      const std::string&, const Envoy::Json::Object&,
+      istio::authn::JwtPayload* payload);
 };
 
-bool JsonIterCallback::JsonIteratorCallback(const std::string& key,
-                                            const Envoy::Json::Object& obj,
-                                            istio::authn::JwtPayload* payload) {
+bool JsonIterCallback::JsonIteratorCallbackForJwtClaims(
+    const std::string& key, const Envoy::Json::Object& obj,
+    istio::authn::JwtPayload* payload) {
   ENVOY_LOG(debug, "{}: key is {}", __FUNCTION__, key);
```

Comment:
Please remove this debug, and many similar. This make debugging actually harder as you need to dig through so many logs.

Reply:
Done.